R/utils.R

```diff
@@ -184,13 +184,14 @@ unescape <- function(str, q="`") {
 
 # convert an XML match into a Lint
 xml_nodes_to_lint <- function(xml, source_file, message, linter,
-                              type = c("style", "warning", "error")) {
+                              type = c("style", "warning", "error"),
+                              offset = 0L) {
```

Comment:
nice extension! eventually i think we can refactor a lot of the XPath lints to use this helper. Basically as long as `message` is static (e.g. this isn't the case for `seq_linter`) I think this can be used.

Reply:
Maybe we can even vectorize on message?
The T and F symbol linter needs a wrapper at the call site that just generates the proper message.

Reply:
Let's tuck this away to revisit for now